config.json

```diff
@@ -1,18 +1,17 @@
 {
   "slug": "r",
   "language": "R",
-  "repository": "https://github.com/exercism/xr",
-  "active": false,
+  "repository": "https://github.com/morphatic/xr",
```

Comment:
The repository needs to be `exercism/xr`, as that is the canonical repo that the x-api uses to serve problems from.